keyboards/nomu30/nomu30.h

```diff
@@ -15,16 +15,14 @@
  */
 #pragma once
 
+#ifdef KEYBOARD_nomu30_rev1
+    #include "rev1.h"
+#elif KEYBOARD_nomu30_rev2
+    #include "rev2.h"
+#endif
+
 #include "quantum.h"
 
-/* This a shortcut to help you visually see your layout.
- *
- * The first section contains all of the arguments representing the physical
- * layout of the board and position of the keys.
- *
- * The second converts the arguments into a two-dimensional array which
- * represents the switch matrix.
- */
 #define LAYOUT( \
```

Comment:
LAYOUT macro should be moved to `rev1.h`

Reply:
Moved 336a70c